pkg/operator/startupmonitorreadiness/readiness_checks.go

```diff
@@ -279,11 +276,20 @@ func goodHealthzEtcdEndpoint(client *http.Client, rawURL string) func(context.Co
 func doHTTPCheckAndTransform(ctx context.Context, client *http.Client, rawURL string, checkName string, httpCheckFn func(ctx context.Context, client *http.Client, rawURL string) (int, string, error)) (bool, string, string) {
 	statusCode, response, err := httpCheckFn(ctx, client, rawURL)
 	if err != nil {
+		if utilnet.IsConnectionRefused(err) {
+			return false, "NetworkError", fmt.Sprintf("waiting for kube-apiserver static pod to listen on port 6443: %v", err)
```

Comment:
is this only for SNO?

Reply:
yes - `readiness_checks` apply on an `sno` cluster

Reply:
technically this can run in HA too.